serve/serve.go

```diff
@@ -95,6 +98,7 @@ func Serve(opts *Options) {
 		})
 	}
 
+	stats.Start(context.Background(), opts.Logger)
```

Comment:
nit: maybe add a log line that stats heartbeat collector started?

Reply:
Good point, done in https://github.com/cloudquery/cq-provider-sdk/pull/237/commits/3b394523d4b57c54e74f9e92eac122db777b7e76